crates/gpui/src/platform/windows/window.rs

```diff
@@ -189,7 +189,6 @@ impl WindowsWindowInner {
     }
 
     fn handle_msg(&self, msg: u32, wparam: WPARAM, lparam: LPARAM) -> LRESULT {
-        log::debug!("msg: {msg}, wparam: {}, lparam: {}", wparam.0, lparam.0);
```

Comment:
Unlike the original diff, this is a diff that completely erases, is this intentional?
In fact, I considered erasing it completely because I wasn't sure if this log would be useful in the future, so if that is the intention, that's fine.

Reply:
Yes!